### PR TITLE
BoltNineHomebrew; The King's Army - Complete Monster Pack

### PR DESCRIPTION
**Pull request by p4535992**
Added monster pack from BoltNineHomebrew; The King's Army - Complete Monster Pack

### CODE REVIEW

File: creature/BoltNineHomebrew; The King's Army - Complete Monster Pack.json [hunk 1]
```diff
@@ -0,0 +1,2563 @@
...
+			"wis": 14,
+			"cha": 18,
+			"passive": 15,
+			"cr": "8",
+			"shortName": "Queen Adria Durron",
+			"fluff": {
+				"entries": [
+					"The King And the Queen",
+					"King Paul Shapechanger and Queen Adria Durron lead the kingdom. Though Paul is King, he is actually not a royal, having married then Princess Adria Durron many years ago. Since the death of Adria’s father, King Patil took the throne besides his wife.",
+					"QUEEN ADRIA DURRON",
```
Comment (TheGiddyLimit) [inline creature/BoltNineHomebrew; The King's Army - Complete Monster Pack.json:64]: You could use some proper header formatting in the fluff; up to you.

File: creature/BoltNineHomebrew; The King's Army - Complete Monster Pack.json [hunk 2]
```diff
@@ -0,0 +1,2563 @@
...
+				{
+					"name": "Spellcasting",
+					"headerEntries": [
+						"Spellcasting Ability=INT",
+						"Global Magic Attack Modifier=12",
+						"Magic Caster Level=16",
+						"Spell Save DC Mod=20",
+						"Nix Riversong Is a 20th-level spellcaster. Its spellcasting ability is Intelligence (spell save {@dc 20}, +12 to hit with spell attacks}. Nix Riversong has the following wizard spells prepared:",
+						"Cantrips (at will): chill touch, dancing lights, frostbite. gust. ray of frost",
+						"1st level (4 slots): absorb elements, charm person, fog cloud, mage armor",
```
Comment (TheGiddyLimit) [inline creature/BoltNineHomebrew; The King's Army - Complete Monster Pack.json:599]: Duplicated spell list with below; remove the one from `headerEntries`

File: creature/BoltNineHomebrew; The King's Army - Complete Monster Pack.json [hunk 3]
```diff
@@ -0,0 +1,2563 @@
...
+				{
+					"name": "Spellcasting",
+					"headerEntries": [
+						"Spellcasting Ability=CHA",
+						"Global Magic Attack Modifier=9",
+						"Magic Caster Level=12",
+						"Spell Save DC Mod=17",
+						"Sane is a 12th-level spelicaster. Its spellcasting ability is Charisma (spell save {@dc 17}, +9 to hit with spell attacks). Sane has the following warlock spells prepared:",
+						"Cantrips (at will): control flame, eldritch blast, message,friends",
+						"1st bevel (4 slots): charm person, fog cloud, mage armor,sleep",
```
Comment (TheGiddyLimit) [inline creature/BoltNineHomebrew; The King's Army - Complete Monster Pack.json:1241]: Another duplicated list

File: creature/BoltNineHomebrew; The King's Army - Complete Monster Pack.json [hunk 4]
```diff
@@ -0,0 +1,2563 @@
...
+			},
+			"spellcasting": [
+				{
+					"name": "Spellcasting",
+					"headerEntries": [
+						"Spellcasting Ability=CHA",
+						"Global Magic Attack Modifier=8",
+						"Magic Caster Level=10",
+						"Spell Save DC Mod=16",
+						"The blue elf is a 10th level spellcaster. His spellcasting ability is Charisma (spell save {@dc 16}, +8 to hit with spell attacks). The blue elf has the following sorcerer spells prepared:",
```
Comment (TheGiddyLimit) [inline creature/BoltNineHomebrew; The King's Army - Complete Monster Pack.json:1542]: Another duplicated list

File: creature/BoltNineHomebrew; The King's Army - Complete Monster Pack.json [hunk 5]
```diff
@@ -0,0 +1,2563 @@
...
+			"spellcasting": [
+				{
+					"name": "Spellcasting",
+					"headerEntries": [
+						"Spellcasting Ability=INT",
+						"Global Magic Attack Modifier=5",
+						"Magic Caster Level=5",
+						"Spell Save DC Mod=13",
+						"The war mage is a 5th level spellcaster. His spellcasting ability is Intelligence (spell save {@dc 13}, +5 to hit with spell attacks). The war mage has the following wizard spells prepared:",
+						"Cantrips (at will): firebolt, mage hand, message,prestidigitation",
```
Comment (TheGiddyLimit) [inline creature/BoltNineHomebrew; The King's Army - Complete Monster Pack.json:2126]: Another duplicated list

Comment (p4535992) [reply creature/BoltNineHomebrew; The King's Army - Complete Monster Pack.json]: Hi removed duplicate spell list, when i have time i will try to beautify the fluffy description ^^

**Pull request merged**